bower.json

```diff
@@ -9,5 +9,9 @@
     "components",
     "spec",
     "debug"
-  ]
+  ],
+  "dependencies": {
+    "leaflet": "~1.0.0",
```

Comment:
until `1.0.0` final is released, lets go with `"leaflet": "1.0.0-beta.2"` explictly.